specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/preview/2016-01-01/Configuration.json

```diff
@@ -0,0 +1,96 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-01-01",
+    "title": "AzureBridgeAdminClient",
+    "description": "AzureBridge Admin Client."
+  },
+  "host": "adminmanagement.local.azurestack.external",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/providers/Microsoft.AzureBridge.Admin/activations/{activationName}/configuration": {
+      "get": {
+        "x-ms-examples": {
+          "Return product name.": {
+            "$ref": "./examples/Configuration/Read.json"
+          }
+        },
+        "description": "Return azure stack configuration data.",
+        "tags": [
+          "Configuration"
+        ],
+        "operationId": "Configuration_Get",
```

Comment:
`Configurations_Get`

Reply:
It gets a single configuration. Not list of them
Why would it be "Configurations"?

Reply:
I think the plural is the typical norm for all the ARM resources

Reply:
The noun defines the operation group name, not the thing(s) the verb returns.  It's plural so that the response model name can be singular else the names collide.  In this case there is no collision.  Is the configuration really singular?  If so then I'm fine with leaving it as singular assuming there isn't another operation group with the same name.

Reply:
there are no other configuration getters and yes, it's really singular)